src/Native/pkg/dir.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <!-- common properties that point to output of native build.  Can be overriden in case of 
          a coordinating build that produces all packages from OS-specific native builds -->
-    <BuildNativePath Condition="'$(BuildNativePath)' == ''">$(BinDir)native\</BuildNativePath>
+    <BuildNativePath Condition="'$(BuildNativePath)' == ''">$(BinDir)$(OSGroup).$(PackagePlatform).$(ConfigurationGroup)\Native\</BuildNativePath>
```

Comment:
It'd be nice to combine "$(OSGroup).$(PackagePlatform).$(ConfigurationGroup)" into a separate property that can be used / re-used.  I don't know that we are using this pattern anywhere, but it seems cleaner.